src/tests/ftest/util/test_utils_container.py

```diff
@@ -368,6 +369,103 @@ def create(self, uuid=None, con_in=None, acl_file=None):
         self.uuid = self.container.get_uuid_str()
         self.log.info("  Container created with uuid %s", self.uuid)
 
+    @fail_on(DaosApiError)
+    def create_snap(self, pool=None, cont=None, snap_name=None, epoch=None,
+                    svc=None, sys_name=None):
+        """Create Snapshot using daos utility
+
+        Args:
+            pool (str): pool uuid
+            cont (str):Cont uuid
+            snap_name (str, optional): Snapshot name
+            epoch (str, optional): Epoch ID
+            svc (str, optional): Pool svc
+            sys_name (str, optional): system name
+        """
+        self.log.info("Creating Snapshot for Container: %s", self.uuid)
+        if not pool:
+            pool = self.pool
+        if not cont:
+            cont = self.uuid
+        if not svc:
+            svc = ",".join(str(rank) for rank in self.pool.svc_ranks)
+        if not sys_name:
+            sys_name = self.pool.name.value
+        if self.control_method.value == self.USE_DAOS and self.daos:
+            # create snapshot using daos utility
+            kwargs = {
+                "pool" : pool.uuid,
+                "cont" : cont,
+                "snap_name" : snap_name,
+                "epoch" : epoch,
+                "svc" : svc,
+                "sys_name": sys_name,
+            }
+            self._log_method("daos.container_create_snap", kwargs)
+            output = self.daos.get_output("container_create_snap", **kwargs)[0]
+
+        elif self.control_method.value == self.USE_DAOS:
+            self.log.error("Error: Undefined daos command")
+
+        else:
+            self.log.error(
+                "Error: Undefined control_method: %s",
+                self.control_method.value)
+
+        self.epoch = output.split()[1]
+
+    @fail_on(DaosApiError)
+    def destroy_snap(self, pool=None, cont=None, snap_name=None, epoch=None,
```

Comment:
Same comment as above. It doesn't make sense to provide this method the ability to do destroy snapshot work with whatever pool or container the caller specifies. That feature is already provided by container_destroy_snap method. The methods in this class should deal with the particular TestContainer object.

Reply:
Done